### DISCUSSION TITLE
go1.20.linux-amd64.tar.gz 对应的gcc版本是？

### DISCUSSION THREAD

**Discussion by zorn-zhao** [labring/sealos#3425] [Q&A] [answered]
yum -y install gcc-C++-x86 64-linux-gnu 提示没有软件包，自己安装gcc 9.3 还提示 cgo: C compiler "x86_64-linux-gnu-gcc" not found: exec: "x86_64-linux-gnu-gcc": executable file not found in $PATH you need install gnu-gcc, like: apt-get install build-essential or yum -y install gcc-c++-x86_64-linux-gnu

**Comment by zorn-zhao** [comment, marked as answer]
gcc 9.3 https://static.kancloud.cn/zheng123/linuxgz/2689163

**Reply by bxy4543** [reply]
execute `gcc --version` to ensure that the installation and version are correct
